src/topics/yaml-guide.md

```diff
@@ -25,7 +25,7 @@ You can skip this section if you are already comfortable with YAML.
 
 Fundamentally, a file written in YAML consists of a set of _key-value pairs_.
 Each pair is written as `key: value`,
-where whitespace after the `:` is optional.
+where whitespace after the `:` is optional, but is required when defining fields in a CWL file.
```

Comment:
In YAML, when defining entries in a map/dictionary, the space after the colon is mandatory, so the original text is in error!

https://yaml.org/spec/1.2.2/#21-collections

```suggestion
where whitespace after the `:` is required.
```